### DISCUSSION TITLE
Delete all RAW files and keep the JPG

### DISCUSSION THREAD

**Discussion by dinisfc** [photoprism/photoprism#4141] [Q&A] [answered]
I imported a lot of photos in both the jpg format and cr3 format (Canon's RAW format). I have no interest in keeping the RAW files and I would like to delete all .cr3 files. Is there any easy way to delete all files of this type?

**Comment by lastzero** [comment, marked as answer]
https://askubuntu.com/questions/377438/how-can-i-recursively-delete-all-files-of-a-specific-extension-in-the-current-di
